test/Deployment/data/WindowsAppRuntime.Test.Main/WindowsAppRuntime.Test.Main.vcxproj

```diff
@@ -87,95 +87,23 @@
-    <NMakePreprocessorDefinitions>WIN32;NDEBUG;$(NMakePreprocessorDefinitions)</NMakePreprocessorDefinitions>
-    <OutDir>$(OutDir)</OutDir>
-    <IntDir>$(IntDir)</IntDir>
-    <NMakeIncludeSearchPath>$(SolutionDir)tools</NMakeIncludeSearchPath>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
-    <NMakeBuildCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) WindowsAppSdkBuildPipeline=$(WindowsAppSdkBuildPipeline) </NMakeBuildCommandLine>
-    <NMakeOutput>$(OutputPath)Deployment.WindowsAppRuntime.Test.Main</NMakeOutput>
-    <NMakeCleanCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) WindowsAppSdkBuildPipeline=$(WindowsAppSdkBuildPipeline) clean</NMakeCleanCommandLine>
-    <NMakeReBuildCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) WindowsAppSdkBuildPipeline=$(WindowsAppSdkBuildPipeline) rebuild</NMakeReBuildCommandLine>
-    <NMakePreprocessorDefinitions>NDEBUG;$(NMakePreprocessorDefinitions)</NMakePreprocessorDefinitions>
-    <OutDir>$(OutDir)</OutDir>
-    <IntDir>$(IntDir)</IntDir>
-    <NMakeIncludeSearchPath>$(SolutionDir)tools</NMakeIncludeSearchPath>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">
-    <NMakeBuildCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) WindowsAppSdkBuildPipeline=$(WindowsAppSdkBuildPipeline) </NMakeBuildCommandLine>
-    <NMakeOutput>$(OutputPath)Deployment.WindowsAppRuntime.Test.Main</NMakeOutput>
-    <NMakeCleanCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) WindowsAppSdkBuildPipeline=$(WindowsAppSdkBuildPipeline) clean</NMakeCleanCommandLine>
-    <NMakeReBuildCommandLine>nmake /C /f makefile.mak ProjectDir=$(ProjectDir) SolutionDir=$(SolutionDir) OutDir=$(OutDir) TargetName=$(TargetName) WindowsAppSdkBuildPipeline=$(WindowsAppSdkBuildPipeline) rebuild</NMakeReBuildCommandLine>
-    <NMakePreprocessorDefinitions>NDEBUG;$(NMakePreprocessorDefinitions)</NMakePreprocessorDefinitions>
-    <OutDir>$(OutDir)</OutDir>
-    <IntDir>$(IntDir)</IntDir>
-    <NMakeIncludeSearchPath>$(SolutionDir)tools</NMakeIncludeSearchPath>
-  </PropertyGroup>
-  <ItemDefinitionGroup>
-  </ItemDefinitionGroup>
   <ItemGroup>
-    <Xml Include="appxmanifest.xml" />
+    <MakeMsixInputAssets Include="$(MSBuildThisFileDirectory)\Assets\*" />
+    <MakeMsixInputs Include="$(MSBuildThisFileDirectory)appxmanifest.xml" />
   </ItemGroup>
+  <Import Project="$(RepoRoot)\MakeMsix.targets" />
```

Comment:
I love the use of RepoRoot here.  Should we consider moving the MakeMsix.targets file into the build directory to keep the root tidy?

Reply:
RepoRoot is pretty neat. I'd been using SolutionDir but RepoRoot is better (especially as we have multiple solutions, only 1 of them in the root dir). Will make some updates.

We've got a bunch other build files in the root. Yes, we probably should shuffle some things, but we have some matters to iron out before we go there.

`build` historically was used by the _build pipeline_ not the local VS content (i.e. not by WindowsAppRuntime.sln etc). I suspect it's useful to have a clear distinction between those things, though that could be separate dirs e.g. `build\pipeline` vs `build\vs`.

We've had some discussions of file layout and some planned improvements. I think it's good for now and will follow up on the more holistic layout / cleanup.